bign256/src/lib.rs

```diff
@@ -132,7 +132,7 @@ impl FieldBytesEncoding<BignP256> for U256 {
 #[cfg(feature = "arithmetic")]
 pub type NonZeroScalar = elliptic_curve::NonZeroScalar<BignP256>;
 
-/// BIGN P-256 public key.
+// /// BIGN P-256 public key.
 // #[cfg(feature = "arithmetic")]
 // pub type PublicKey = elliptic_curve::PublicKey<BignP256>;
```

Comment:
Sidebar: hmm, why's this commented out

Reply:
I have no idea, that got added in #1046
The comment on the documentation was added by rustfmt.